src/index.jsx

```diff
@@ -32,8 +37,10 @@ if (returnPath) {
   store.dispatch(push(returnPath));
 }
 
-store.dispatch(fetchAccounts());
-store.dispatch(fetchRoles());
+store.dispatch(fetchDatasets());
```

Comment:
Why are these actions dispatched here? Should't  the datasets container component be responsible for some like dispatching the fetchDatasets?

Reply:
you need the schema, and ui schemas and all datasets anyway